apps/web/src/pages/profile/[user].tsx

```diff
@@ -168,20 +168,31 @@ const ProfilePage: NextPageWithLayout<ProfileProps> = ({ userAddress }) => {
 
           {hasDaos && (
             <>
-              <Grid columns={isMobile ? 1 : 3} gap={'x12'}>
-                {tokens?.tokens.map((x, i) => (
-                  <Link
-                    key={i}
-                    href={`/dao/${x.tokenContract?.collectionAddress}/${x.tokenId}`}
-                  >
-                    <TokenPreview name={x.name} image={x.image?.url} />
-                  </Link>
-                ))}
-              </Grid>
+              {(tokens?.tokens.length || 0) > 0 ? (
```

Comment:
can this be `!!tokens?.tokens.length` i think that `0` is falsy and so `!!0 === true`